src/main/java/com/indeed/operators/rabbitmq/reconciliation/rabbitmq/ShovelReconciler.java

```diff
@@ -39,8 +39,8 @@ public void reconcile(final RabbitMQCluster cluster) {
 
             Preconditions.checkNotNull(secret, String.format("Could not find secret with name [%s] in namespace [%s]", destSecretName, destSecretNamespace));
 
-            final String username = secretsController.decodeSecretPayload(secret.getData().get("default-username"));
-            final String password = secretsController.decodeSecretPayload(secret.getData().get("default-password"));
+            final String username = secretsController.decodeSecretPayload(secret.getData().get("username"));
```

Comment:
I haven't really been paying attention but it seems like I've seen these changes at least three or four times already.  Should these be constants?

Reply:
"username" and "password", I mean.

Reply:
yea probably. stupid laziness!

Reply:
Thanks!